packages/app-layout/AppLayout.tsx

```diff
@@ -0,0 +1,161 @@
+import { h, Component, prop } from 'skatejs';
+
+import {
+  GenericTypes,
+  matchMedia
+} from '../_helpers';
+
+import { HamburgerButton } from './components/HamburgerButton';
+import { Drawer } from './components/Drawer';
+import { Overlay } from './components/Overlay';
+import { Nav } from './components/Nav';
+import { NavContent } from './components/NavContent';
+
+import styles from './AppLayout.scss';
+
+const DEFAULT_RESPONSIBLE_WIDTH = '640px';
```

Comment:
I'm following polymer [way](https://github.com/PolymerElements/app-layout/blob/master/app-drawer-layout/app-drawer-layout.html#L172) and yes you don't always want to use px let's let developer to choose.